common/environment/environment.cpp

```diff
@@ -168,8 +172,8 @@ class CLocalEnvironment : public CInterface, implements IConstEnvironment
     unsigned getNumberOfDropZonesByComputer(const char * computer) const
     {
         buildDropZoneCache();
-        StringBuffer xpath("@");
-        xpath.append(computer).append(NUMBER_OF_DROPZONES_SUFFIX);
+        StringBuffer xpath;
+        createComputerNameXpath(computer, xpath);
         unsigned numOfDropZoneByComputer = numOfDropzonesByComputer->getPropInt(xpath, -1);
         return (numOfDropZoneByComputer != -1 ? numOfDropZoneByComputer : 0);
```

Comment:
Is there any reason to default to -1 then check it?
In the absence of it being set it would return 0 anyway..
So the above 2 lines is equivalent to: 
```
return numOfDropzonesByComputer->getPropInt(xpath);
```